src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningSummary.cshtml

```diff
@@ -163,19 +144,25 @@
             </dl>
 
             <p class="govuk-body">
-                If the figures you have entered are correct then you need to re-check your funding calculation for the minimum price reduction due to RPL to ensure it follows the funding rules.
+                The formula for the calculation above is outlined in the <a class="govuk-link" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1155957/Apprenticeship_funding_rules_2324_Version_1.pdf">apprenticeship
+                funding rules</a>.
             </p>
 
-                <input type="hidden" name="answers-checked" value="true">
-                <button class="govuk-button govuk-!-margin-top-6" data-module="govuk-button">
-                    Save and Continue
-                </button>
-                <br>
-            </form>
+            <p class="govuk-body">
+                If the figures you have entered are correct your organisation needs to adjust it's calculation method to make it compliant with the funding rules.
```

Comment:
There should be no apostrophe in `it's` (though this matches the content in the ticket).

Reply:
Fixed Chris, thanks